config/conda_build_config.yaml

```diff
@@ -1,6 +1,6 @@
 c_compiler_version:
```

Comment:
Do we need these anymore? It looks like we are now using the same compiler versions as the rest of Open-CE.

Reply:
Yes, correct! We don't need this anymore now. I'll remove the file.